src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -116,6 +132,14 @@ public void stop() {
     public void end() {
         timer.cancel();
         timer = new Timer();
+
+        borderTimeOffsetTime.cancel();
+        borderTimeOffsetTime = new Timer();
+
+        final var worldBorder = server.getOverworld().getWorldBorder();
+        // Stops the border shrinking.
+        worldBorder.setSize(worldBorder.getSize());

```

Comment:
`worldBorder.setSize(Molehunt.CONFIG.getInitialWorldSize())` should be better

Reply:
no, the idea here is to stop it where it's currently at